test_conformance/api/test_kernel_private_memory_size.cpp

```diff
@@ -0,0 +1,58 @@
+#include "harness/errorHelpers.h"
```

Comment:
Might need a license comment at the top of this file

Reply:
Added a license.